index.go

```diff
@@ -207,9 +212,15 @@ type DocIDReader interface {
 	Close() error
 }
 
+type FetchIOStats interface {
```

Comment:
Don't see the need for this interface, replace with `BytesRead() uint64` method in all the other interfaces.

Reply:
Done.